pkg/specgen/generate/container.go

```diff
@@ -481,7 +489,68 @@ func ConfigToSpec(rt *libpod.Runtime, specg *specgen.SpecGenerator, contaierID s
 		}
 	}
 	specg.OverlayVolumes = overlay
-	specg.Mounts = conf.Spec.Mounts
+	name, mounts := c.SortUserVolumes(c.Spec())
+	tempInspect, err := c.GetInspectMounts(name, conf.ImageVolumes, mounts)
```

Comment:
Hate this function name GetInspectMounts... Nothing to fix though.

Reply:
if we are changing `GetMounts` to `Mounts` above I can rename this `GetMounts`